api/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -126,18 +126,20 @@ public abstract class LoadBalancer {
    *
    * @param servers the resolved server addresses, never empty.
    * @param attributes extra information from naming system.
+   * @return {@code true} if the resolved addresses were accepted. {@code false} if rejected.
    * @deprecated override {@link #handleResolvedAddresses(ResolvedAddresses) instead}
    * @since 1.2.0
    */
   @Deprecated
-  public void handleResolvedAddressGroups(
+  public boolean handleResolvedAddressGroups(
```

Comment:
Let's not change this one. We can double-check how long this has been deprecated, but I suspect a very long time. We may want to delete it to make this migration easier.

Reply:
I went with deleting it. Because the current one gets deprecated in this PR I figured having one deprecated version of this is enough.